Win32Emu.Wasm/Backend/WasmRenderingBackend.cs

```diff
@@ -45,8 +59,10 @@ public bool Initialize(int width, int height, string title = "Win32Emu Display")
 			
 			_logger.LogInformation("[WASM] Initializing rendering backend ({Width}x{Height})", width, height);
 			
-			// Initialize canvas through JavaScript
-			_jsRuntime.InvokeVoidAsync("initializeEmulator", _canvasId).AsTask().Wait();
+			// Fire-and-forget JS initialization. See class remarks for WASM-specific behavior.
+			// IsInitialized will be true before JS completes - this is intentional and safe
+			// because the Blazor component awaits initializeEmulator before starting emulation.
+			_ = _jsRuntime.InvokeVoidAsync("initializeEmulator", _canvasId);
```

Comment:
Call await on this, and maybe return the result? Enclosing method will need to be Async too